src/runtime-rs/src/shim/Cargo.toml

```diff
@@ -2,32 +2,41 @@
 name = "shim"
 version = "0.1.0"
 authors = ["The Kata Containers community <kata-dev@lists.katacontainers.io>"]
+description = "Containerd shim v2 runtime for Kata Containers"
+keywords = ["kata containers", "shimv2"]
+#categories = [""]
+repository = "https://github.com/kata-containers/kata-containers.git"
+#readme = "README.md"
+license = "Apache-2.0"
```

Comment:
Why not put all this information in when the file is first created.

Reply:
Fixed in the second revision: rebased the patches by logical separations